public/js/components/notification-list.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { Component, PropTypes } from 'react/addons';
+import Notification from 'components/notification';
+
+var ReactCSSTransitionGroup = React.addons.CSSTransitionGroup;
+
+export default class NotificationList extends Component {
+
+  static propTypes = {
+    TransitionGroup: PropTypes.node,
+    notifications: PropTypes.array,
+    removeNotification: PropTypes.func.isRequired,
+  }
+
+  static defaultProps = {
+    TransitionGroup: ReactCSSTransitionGroup,
```

Comment:
I think `notifications` should be set to an empty array by default here, right? Or maybe it should be a required prop?